### PR TITLE
Ins 792 geojson

### PR DESCRIPTION
**Pull request by AmirShayegh**

### CODE REVIEW

File: app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts [hunk 1]
```diff
@@ -326,7 +327,8 @@ export class MapPreviewComponent implements OnInit, AfterViewInit, AfterViewChec
    * of the border of BC
    */
   get bcBorderCoordinates(): MapMarker[] {
-    const coordinates = this.bcGeoJSON.features[0].geometry.coordinates;
+    const coordinates = this.bcGeoJSON.features[0].geometry.coordinates[0][0];
+    console.dir(coordinates);
```
Comment (andrea-williams) [inline app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts]: do we still need to log these coords in the console?

File: app/lucy/src/app/services/sso.service.ts [hunk 2]
```diff
@@ -368,6 +371,8 @@ export class SsoService {
       this.code = codeFromRoute;
     }
 
+    console.log("\nhere")
```
Comment (andrea-williams) [inline app/lucy/src/app/services/sso.service.ts]: some more console.log calls in this file that probably aren't needed anymore

File: app/lucy/src/app/constants/app-constants.ts [hunk 3]
```diff
@@ -24,8 +24,8 @@ export class AppConstants {
 
     // SSO
     static SSOConstants = {
-        SSO_CLIENT_ID : `lucy`,
-        SSO_BASE_URL : `https://sso.pathfinder.gov.bc.ca`,
+        SSO_CLIENT_ID : `invasives-bc`,
+        SSO_BASE_URL : `https://sso-dev.pathfinder.gov.bc.ca`,
```
Comment (fwpushan) [inline app/lucy/src/app/constants/app-constants.ts]: Remove console log

File: app/lucy/src/app/services/sso.service.ts [hunk 4]
```diff
@@ -353,7 +353,10 @@ export class SsoService {
    * then clear query parameters.
    */
   private async handleLoginOnRedirect(): Promise<boolean> {
+    console.log("** We were re-directed... getting code from route\n")
     const codeFromRoute = this.extractCodeFromRoute();
+    console.log("** Code: \n")
+    console.log(codeFromRoute);
```
Comment (fwpushan) [inline app/lucy/src/app/services/sso.service.ts]: Remove console log

File: app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts (hunk 1 above)
Comment (AmirShayegh) [reply app/lucy/src/app/components/Utilities/map-preview/map-preview.component.ts]: good catch

**Pull request merged**